client/checkout/classic/payment-processing.js

```diff
@@ -274,6 +274,14 @@ async function createStripePaymentElement(
 		fonts: getFontRulesFromPage(),
 	};
 
+	if ( paymentMethodType === 'wechat_pay' ) {
+		options.paymentMethodOptions = {
+			wechatPay: {
+				client: 'web',
+			},
+		};
+	}
```

Comment:
It looks like this isn't necessary, since the server is sending this option directly when creating the intent.